subprojects/common/report-viewer/src/main/kotlin/com/avito/report/model/TestName.kt

```diff
@@ -20,24 +20,22 @@ data class TestName(val className: String, val methodName: String) {
 val TestName.team: Team
     get() = when {
         packageName.startsWith(domofondPrefix) -> Team("domofond")
-        packageName.startsWith(avitoPrefix) -> {
+        packageName.startsWith(avitoPrefix) ->
             try {
                 val unitPrefix = packageName.substringAfter("$avitoPrefix.").substringBefore(delimiter)
                 Team(unitPrefix.replace("_", "-"))
             } catch (e: Exception) {
                 Team.UNDEFINED
             }
-        }
         else -> Team.UNDEFINED
     }
 
 // todo убрать определение features из пакета, будут явно указаны в аннотации к тесту
 val TestName.features: List<String>
     get() = when {
-        packageName.startsWith(domofondPrefix) -> {
+        packageName.startsWith(domofondPrefix) ->
             packageName.substringAfter("$domofondPrefix.")
                 .split(delimiter)
-        }
```

Comment:
I have doubts. I find it's harder to read in cases with long chains.

Reply:
I understood the reason behind that rule as you misguided, especially in big blocks of code, that it's not a single expression.

Also encourages you to extract functions

Reply:
> Also encourages you to extract functions

Yeah, this is a good argument for it.